pom.xml

```diff
@@ -925,6 +925,14 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
+                </configuration>
+            </plugin>
```

Comment:
Is this needed?

Reply:
Uggh, thanks for catching that on your vacation. It's probably one of those that I inadvertently pressed IntelliJ's suggestion and just got lost in the changes. I will roll that back.

Reply:
Rolled back